app/models/user.rb

```diff
@@ -74,12 +74,12 @@ def self.create_or_update_from_hash(hash)
     user = User.where(external_id: hash[:external_id].to_s).first || User.new
 
     # attributes are always a string hash
-    attributes = user.attributes.merge(hash.stringify_keys) do |key, old, new|
-      if key == 'role_id'
-        if !User.exists? # first user will be the super admin
-          Role::SUPER_ADMIN.id
+    attributes = user.attributes.merge(hash.stringify_keys) do |attribute, old, new|
+      if attribute == 'role_id'
+        if !User.where.not(email: 'seed@example.com').exists?
```

Comment:
FYI @yizhang-zen updated this to be even more efficient ... only a single query and not actually reading any data from the db